src/Components/Display.js

```diff
@@ -0,0 +1,43 @@
+import React,{useState} from 'react';
+import axios from 'axios';
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import "./display.css"
+
+const Display = (props) => {
+    const [state,setState]=useState("")
+    const navigate=useNavigate();
+    
+    useEffect(() => {
+        axios.get('http://localhost:5000/display/'+props.email)
```

Comment:
Why did you choose to add the email in the url instead of passing it as a body param?